### PR TITLE
Create Mentor Page

### LINKED ISSUE #168: 💡 [REQUEST] - Create A mentor page
see -
https://mentro.tech/mentor

We want a mentor page....this would be nice issue

### PR DESCRIPTION
**Pull request by Isha988**
## Related Issue
Closes #168 

## Description of Changes
Started building Mentor page

## Checklist:

- [ ] My code adheres to the established style guidelines of this project.
- [ ] I have conducted a self-review of my code.
- [ ] I have included comments in areas that may be difficult to understand.
- [ ] I have made corresponding updates to the project documentation.
- [ ] My changes have not introduced any new warnings.

## Screenshots

|           Original           |           Updated           |
| :--------------------------: | :-------------------------: |
| Original screenshot | Updated screenshot |

Please provide any necessary screenshots to illustrate the changes made.

### CODE REVIEW

**Comment by Isha988** [comment]
Hey, @anmode  and @Abhinavcode13 , I have started building mentor page . Its route is '/mentor'.
One request, would you please provide me the graphics and content to add  on the page.

**Comment by anmode** [comment]
So @Isha988 folder structure be like..... mentorR.page.js.  actually we have currently `mentor.page.js` it should be `mentorProfile.page.js`. We will change is meanwhile you create a page `mentorR.page.js.`. and create a folder in component as `mentorPage` store all the components of this page in it .
hope you get that.

**Comment by anmode** [comment]
For the content shortly I'll update in this pR. Please If anyone collaborate with you on this pR to create graphics. 
Okay give me time I'll update graphics.

**Comment by Isha988** [comment]
> So @Isha988 actually we have currently `mentors.page.js` it should be `mentorsList.page.js`. We will change, you created page `mentor.page.js.` that's cool and please create a folder in component as `mentorPage` store all the components of this page in it .

Okay, I will

**Comment by Isha988** [comment]
@anmode I have added a new section at the last and also updated the companies logo, please have a look at that. 
And if you want any new sections or any updates so please let me know.

File: components/mentorPage/Company.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        <img src={imgSrc} height={30} />
+      </div>
+      <div className="col d-none d-lg-block text-left p-0">
+        <p className="font-weight-bold text-muted m-0">{name}</p>
+      </div>
+    </div>
+  );
+}
+
+export default Company;
```
Comment (anmode) [inline components/mentorPage/Company.js:16]: compony <-----> Intern

**Review by anmode — APPROVED** [review]
Impressive.
1. FAQ Section needed
2. No worries for the content we will add that.
3. `png` to `svg` or `webp` if possible

**Comment by Isha988** [comment]
> Impressive.
> 
> 1. FAQ Section needed
> 2. No worries for the content we will add that.
> 3. `png` to `svg` or `webp` if possible

Okay, Will create a FAQ section.
I will search for webp or svg logo files, if I get so I will change them.

**Comment by Abhinavcode13** [comment]
> > Impressive.
> > 
> > 1. FAQ Section needed
> > 2. No worries for the content we will add that.
> > 3. `png` to `svg` or `webp` if possible
> 
> Okay, Will create a FAQ section. I will search for webp or svg logo files, if I get so I will change them.

Cool @Isha988 ;)

**Comment by anmode** [comment]
A update @Isha988 ... we are using this color `845ec2`. make sure to setup theme button color and on click just make it little dark.

**Comment by Isha988** [comment]
> A update @Isha988 ... we are using this color `#845ec2`. make sure to setup theme button color and on click just make it little dark.

Okay

**Comment by Isha988** [comment]
@anmode @Abhinavcode13 I have added the FAQ section, changed company logo images to svg and changed the color of button. Have a look at them

File: components/mentorPage/Company.js (hunk 1 above)
Comment (anmode) [reply components/mentorPage/Company.js:16]: @Isha988 ?

File: components/mentorPage/FAQ.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  const toggleAns = () => setOpen(!open);
+
+  return (
+    <div className="gray-bg mb-20 rounded container py-5 text-justify">
+      <div className="row justify-content-between align-items-center">
+        <p className="h2 col-10 col-sm-11">{ques}</p>
+        <div className="col-1 cursor-pointer" onClick={toggleAns}>
+          {open ? (
+            <i class="fas fa-chevron-up"  />
+          ) : (
```
Comment (anmode) [inline components/mentorPage/FAQ.js:14]: Make it `setOpen` not only click on pointer but to on the whole container....clicking anywhere open

**Review by anmode — APPROVED** [review]
Minor Changes and it is good to merge.. Great @Isha988 .

**Comment by Isha988** [comment]
@anmode Required changes made.

**Comment by Isha988** [comment]
`company.js` contains the company component.

**Review by anmode — COMMENTED** [review]
great!
Some more minor coments:
1. We want to show `our top mentor did interne at ` instean `our top mentor works at`
2. At one time only a faq should be open. for eg. I open first one and then when i open 2nd fag it should be close. ( I am not going in logic for now)

All things are good!

**Comment by Isha988** [comment]
> great! Some more minor coments:
> 
> 1. We want to show `our top mentor did interne at ` instean `our top mentor works at`
> 2. At one time only a faq should be open. for eg. I open first one and then when i open 2nd fag it should be close. ( I am not going in logic for now)
> 3. Important one - Mobile optimization
> 
> All things are good!

Mobile optimization? Does it means responsive?

**Comment by Abhinavcode13** [comment]
YEAH @Isha988

**Comment by Isha988** [comment]
> YEAH @Isha988

I think it's already responsive.

**Comment by anmode** [comment]
> > YEAH @Isha988
> 
> I think it's already responsive.

yeah I know it is reponsive...but not fully optimized. Just check a little bit. Otherwise, I'll note again all optimization and list here. Ok try and address my point 1 and 2 please.
btw really impressed by your work.

**Comment by Isha988** [comment]
> > > YEAH @Isha988
> > 
> > 
> > I think it's already responsive.
> 
> yeah I know it is reponsive...but not fully optimized. Just check a little bit. Otherwise, I'll note again all optimization and list here. Ok try and address my point 1 and 2 please. btw really impressed by your work.

@anmode Made the changes mentioned in points 1 and 2.
For mobile optimization I made few changes have a look at it still if some changes are required so please list them here.

**Review by anmode — APPROVED** [review]
Great, A clean PR. Congrats @Isha988 .

**Pull request merged**